packages/blade/src/components/Tabs/TabsContext.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import type { TabsProps } from './types';
+import { throwBladeError } from '~utils/logger';
+import type { ControllableStateSetter } from '~utils/useControllable';
+
+type TabsContextProps =
+  | (Pick<TabsProps, 'size' | 'variant' | 'isFullWidthTabItem' | 'isLazy'> & {
+      isVertical: boolean;
+      baseId: string;
+      selectedValue: string;
```

Comment:
would it make more sense if we call this as `selectedTab` rather than `selectedValue`?

Reply:
both works but value makes more sense because even the public API our nomenclature is `value` not `selectedTab` 

<TabPanel value="payments" \/>